README.md

```diff
@@ -0,0 +1,93 @@
+# Twitter Driver
+
+An [Membrane.io](https://membrane.io/) driver for Twitter.
+
+## Usage
+
+1. Install the [Membrane VSCode Extension](https://marketplace.visualstudio.com/items?itemName=membrane.membrane).
+2. Setup on extension your [Membrane's CLI binary (mctl)](https://membrane.io/download) path.
+3. Login / Sign up with ```mctl login```.
+4. Update the program on Membrane with the VSCode Command palette `(cmd+shift+p)`\
+  ```> Membrame: Update current program```
+
+## Actions
+
+Configure [APP KEY and APP SECRET](https://developer.twitter.com/en/portal/register/keys)
+
+$~~~~$`mctl action "twitter:configure(APP_KEY: '<string>', APP_SECRET: '<string>')"`
+
+Create a tweet (also support for polls)
+
+$~~~~$`mctl action "twitter:tweet(text: 'Hi!')"`
```

Comment:
```suggestion
$~~~~$`mctl action "twitter:tweet(text:'Hi!')"`
```
Currently grefs can't have whitespace. We can fix that later but let's remove all whitespace for now